templates/components/vectordbs/python/none/vectordb.py

```diff
@@ -8,6 +8,8 @@ def get_vector_store():
     if not os.path.exists(STORAGE_DIR):
         vector_store = SimpleVectorStore()
     else:
-        vector_store = SimpleVectorStore.from_persist_dir(STORAGE_DIR)
+        vector_store = SimpleVectorStore.from_persist_dir(
+            STORAGE_DIR, namespace="default"
+        )
     vector_store.stores_text = True
```

Comment:
Stores_text = true is needed? If so a comment would be good

Reply:
Previously, i used VectorStoreIndex.from_vector_store() so the vector_store must enable the stores_text (but it's not able to load the index). But now i'm using the load_index_from_storage so the stores_text could be redundant, let me check.

Reply:
Just removed, we don't need stores_text = true